### PR TITLE
fix(tool): Added more files to restrict from direct translations

### PR DESCRIPTION
**Pull request by RandellDawson**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

This PR adds the non-English versions of `intro.json` and `translations.json` to the files restricted from PRs coming from forked repos.

### CODE REVIEW

**Comment by RandellDawson** [comment]
Feel free to suggest alternate comment messages.  I was not exactly sure what the message should say.

File: .github/workflows/no-prs-to-translation.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ on:
     paths:
       - 'curriculum/challenges/**'
       - '!curriculum/challenges/english/**'
+      - 'client/i18n/locales/**'
```
Comment (ojeytonwilliams) [inline .github/workflows/no-prs-to-translation.yml:9]:
```suggestion
      - 'client/i18n/locales/**/intro.json'
      - 'client/i18n/locales/**/translations.json'
```
Those are the only two that are handled by Crowdin, right?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: .github/workflows/no-prs-to-translation.yml (hunk 1 above)
Comment (RandellDawson) [reply .github/workflows/no-prs-to-translation.yml]: Oops.  Yes, I meant to do that but for some reason just mimicked the previous two lines.

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM :+1:

File: .github/workflows/no-prs-to-translation.yml [hunk 2]
```diff
@@ -21,13 +24,13 @@ jobs:
                 issue_number: context.issue.number,
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                body: "Thanks for your pull-request. We are no longer accepting curriculum-related changes to non-English files. Please visit [our contributing guidelines](https://contribute.freecodecamp.org) to learn more about translating freeCodeCamp's resources."
+                body: "Thanks for your pull-request. We are no longer accepting changes to the non-English versions of curriculum files or intro.json or translations.json. Please visit [our contributing guidelines](https://contribute.freecodecamp.org) to learn more about translating freeCodeCamp's resources."
```
Comment (raisedadead) [inline .github/workflows/no-prs-to-translation.yml:27]:
```suggestion
                body: "Thanks for your pull-request. We are no longer accepting changes to the non-English versions of files in parts of this codebase. This pull-request seems to change some of those. Please visit [our contributing guidelines](https://contribute.freecodecamp.org) to learn more about translating freeCodeCamp's resources."
```

Let's make it generic instead.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**